diff --git a/node_modules/axios/lib/defaults.js b/node_modules/axios/lib/defaults.js
index 23e1cbd..0b91da2 100644
--- a/node_modules/axios/lib/defaults.js
+++ b/node_modules/axios/lib/defaults.js
@@ -36,7 +36,9 @@ var defaults = {
       utils.isBuffer(data) ||
       utils.isStream(data) ||
       utils.isFile(data) ||
-      utils.isBlob(data)
+      utils.isBlob(data) ||
+      // 放行 PostData
+      (data !== null && typeof data === 'object' && data.toString() === '[object PostData]')
     ) {
       return data;
     }
diff --git a/node_modules/axios/lib/utils.js b/node_modules/axios/lib/utils.js
index e028c67..ce878c5 100644
--- a/node_modules/axios/lib/utils.js
+++ b/node_modules/axios/lib/utils.js
@@ -193,12 +193,18 @@ function trim(str) {
 function isStandardBrowserEnv() {
   if (typeof navigator !== 'undefined' && (navigator.product === 'ReactNative' ||
                                            navigator.product === 'NativeScript' ||
-                                           navigator.product === 'NS')) {
+                                           navigator.product === 'NS' ||
+                                           // 兼容 Taro 3
+                                           // ref: https://github.com/NervJS/taro/commit/7349f716111accb6a39c72ed0f1e6cbc166fa32b
+                                           navigator.product === 'Taro')) {
     return false;
   }
   return (
     typeof window !== 'undefined' &&
-    typeof document !== 'undefined'
+    typeof document !== 'undefined' &&
+    // 兼容 Taro 3
+    // ref: https://github.com/NervJS/taro/issues/5741#issuecomment-601102518
+    !!document.scripts
   );
 }
 
